### PR TITLE
Archive update

### PR DESCRIPTION
**Pull request by ineiti**
Build the images on github.

### CODE REVIEW

File: .github/workflows/build.yml [hunk 1]
```diff
@@ -10,7 +10,8 @@ on:
 jobs:
   build:
     name: Lint, build and test
-    runs-on: ubuntu-18.04
+    runs-on: ubuntu-22.04
+    if: false
```
Comment (PascalinDe) [inline .github/workflows/build.yml:14]: why not remove this if it is never run?

File: .github/workflows/build.yml [hunk 2]
```diff
@@ -76,11 +77,11 @@ jobs:
...
-      - uses: docker/build-push-action@v4
-        with:
-          context: backend
-          push: true
-          tags: c4dt/service-stainless-backend:latest
+#      - uses: docker/build-push-action@v4
+#        with:
+#          context: backend
+#          push: true
+#          tags: c4dt/service-stainless-backend:latest
```
Comment (PascalinDe) [inline .github/workflows/build.yml:84]: please remove the commented lines

File: docker-compose.yml [hunk 3]
```diff
@@ -2,53 +2,59 @@ version: "3"
...
 services:
   conode-1:
-    image: c4dt/service-stainless-backend:latest
+    image: ghcr.io/c4dt/service-stainless-backend:latest
     build:
       context: backend
-    volumes:
-      - ./backend/build/conode-1:/config
+#    volumes:
+#      - ./backend/build/conode-1:/config
```
Comment (PascalinDe) [inline docker-compose.yml:9]: please remove the commented lines

File: docker-compose.yml [hunk 4]
```diff
@@ -2,53 +2,59 @@ version: "3"
...
     build:
       context: webapp
-    network_mode: host
-    volumes:
-      - ./webapp/src/assets:/webapp/assets
+#    network_mode: host
+    ports:
+      - "8080:8080"
+#    volumes:
+#      - ./webapp/src/assets:/webapp/assets
```
Comment (PascalinDe) [inline docker-compose.yml:60]: please remove the commented lines

**Comment by ineiti** [comment]
Sorry, that was not supposed to be reviewed yet. I'm still in the middle of getting this one to work. And it's a mess!

**Pull request closed without merging**